### ISSUE TITLE
Use Cn instead of C2 basis that will be deprecated + cleaning

### ISSUE THREAD

**Issue by ikalash** [sandialabs/LCM#58]
More detail found here: https://github.com/sandialabs/Albany/pull/800 .  We need to do the same thing for LCM as Mauro did for Albany.

**Comment by lxmota** [comment]
@ikalash No, we aren't using any C2 quad elements anywhere that I could find.

**Comment by lxmota** [comment]
Closing as won't fix — irrelevant after recent code changes.
